conf/application.conf

```diff
@@ -79,6 +79,7 @@ aws {
   access.key.id = ${?AWS_KEY}
   secret.access.key = "ENTER YOUR SECRET KEY"
   secret.access.key = ${?AWS_SECRET}
+  service_endpoint = "http://127.0.0.1:8001"
```

Comment:
Do you think it makes sense to put this value into different environments? we will need it to be null for Prod but point at localhost s3 mock for integration tests..? @konradwudkowski